src/mondemand_statdb.erl

```diff
@@ -234,7 +234,7 @@ try_update_counter (InternalKey =
 
 
 -record(md_gcounter,
-        { rate :: integer() | 'undefined',
+        { rate = 0 :: integer() | 'undefined',
```

Comment:
Does dialyzer complain about this line?  Isn’t it always an integer now?

Reply:
You're right, I forgot to update the spec.